components/dashboard/src/projects/NewProject.tsx

```diff
@@ -166,30 +213,10 @@ export default function NewProject() {
                 await getGitpodService().server.triggerPrebuild(project.id, null);
             })();
         }
-    }, [project, sourceOfConfig]);
+    }, [guessedConfigString, project, sourceOfConfig]);
```

Comment:
Reading the code, I think depending on `guessedConfigString` in addition to `sourceOfConfig` has the same effect. No objection here if it makes the linter happy. 🙂

Reply:
Yup, same reasoning as in https://github.com/gitpod-io/gitpod/pull/8995/files#r839619341